apps/mobile/src/components/register-adoption-form/adoption-form.hooks.test.tsx

```diff
@@ -46,3 +54,52 @@ describe('useMultiStepNavigation', () => {
     });
   });
 });
+
+describe('#getStepsByOrder', () => {
```

Comment:
Not a common pattern in jest. It's more a RSpec thing AFAIK

```suggestion
describe('getStepsByOrder', () => {
```